src/Symfony/Component/Form/Tests/Extension/Core/Type/BaseTypeTest.php

```diff
@@ -11,10 +11,12 @@
 
 namespace Symfony\Component\Form\Tests\Extension\Core\Type;
 
+use Symfony\Component\Form\Test\TypeTestCase;
```

Comment:
this is wrong, because it conflicts with the deprecated TypeTestCase being in this namespace. you should either keep the FQCN or alias the imported class (this is the reason I kept the FQCN when switching from the deprecated class to the new one btw)